app/src/main/res/values/strings.xml

```diff
@@ -418,6 +418,9 @@
     <string name="show_channel_details">Show channel details</string>
     <string name="enqueue_stream">Enqueue</string>
     <string name="enqueued">Enqueued</string>
+    <string name="enqueue_next_stream">Enqueue next</string>
+    <string name="enqueued_next">Enqueued next</string>
+    <string name="start_here_on_main">Start playing here</string>
```

Comment:
Why was this string added?
```suggestion
```

Reply:
@Stypox you decided to leave it as it is so that if it ever becomes useful again it's ready to be used  more than 2 years ago here [https://github.com/TeamNewPipe/NewPipe/pull/2368#issuecomment-497330767](url)
I still remember

Reply:
That link seems to give back a 404 error?

Reply:
Thank you for pointing out the error. I have edited it.

Reply:
> I still remember that.. (in a lighter way..)

Impressive.
Gone and forgotten now.